modules/nf-core/propr/propd/meta.yml

```diff
@@ -39,6 +39,9 @@ input:
         description: |
           The value within the contrast_variable column of the sample sheet that
           should be used to derive the target samples
+    - formula:
```

Comment:
propd doesn't support formulas. Do we still want it to accept a formula field for consistency that is ignored, or is it better to remove it?